### PR TITLE
 apply the workaround to enable speedy for IE/Edge

### LINKED ISSUE #1464: speedy injection bugs for styled(StyledComponent) in MS browsers
I don't have a repro just yet, but things are definitely injecting out of order in IE11. Looking at MDN, it seems that IE has a [spotty implementation of `insertRule`](https://developer.mozilla.org/en-US/docs/Web/API/CSSStyleSheet/insertRule) so we might just want to opt out of SPEEDY mode for the IE user agent.

### PR DESCRIPTION
**Pull request by probablyup**
Closes #1464

### CODE REVIEW

**Comment by probablyup** [comment]
cc @mitchellhamilton this might be relevant for emotion

**Comment by mitchellhamilton** [comment]
[emotion already inserts an empty text node into all style elements](https://github.com/emotion-js/emotion/blob/master/packages/create-emotion/src/sheet.js#L51) so I think this is already fixed in emotion.

File: src/models/BrowserStyleSheet.js [hunk 1]
```diff
@@ -164,6 +161,9 @@ if (!DISABLE_SPEEDY) {
         )
       }
 
+      // workaround for an IE/Edge bug: https://twitter.com/probablyup/status/958138927981977600
+      newEl.textContent = ' '
```
Comment (philpl) [inline src/models/BrowserStyleSheet.js]: Lgtm if it works :+1:

File: src/models/BrowserStyleSheet.js [hunk 2]
```diff
@@ -154,7 +152,6 @@ if (!DISABLE_SPEEDY) {
     replaceElement() {
       // Build up our replacement style tag
       const newEl = this.el.cloneNode(false)
-      newEl.setAttribute(SC_ATTR, '')
```
Comment (philpl) [inline src/models/BrowserStyleSheet.js]: Any reason you'd want this to go?

**Comment by probablyup** [comment]
This is good to go 👍

**Pull request merged**